web-test-runner.config.mjs

```diff
@@ -140,6 +140,16 @@ export default {
 			</head>
       <body>
         <script type="module" src="${testFramework}"></script>
+				<script type="module">
+					<!-- Hack to disable Lit dev mode warnings -->
```

Comment:
This needs to be a javascript comment
```suggestion
					// Hack to disable Lit dev mode warnings
```

Reply:
Oh, I saw the problem before your suggestion. So did this already. :-/